paper-menu.html

```diff
@@ -30,41 +65,45 @@
       @apply(--paper-menu);
     }
 
-    :host > ::content > [disabled] {
+    /* need a wrapper element to make this higher specificity than the :host rule in paper-item */
+    .content > ::content > .iron-selected {
+      font-weight: bold;
+
+      @apply(--paper-menu-selected-item);
+    }
+
+    .content > ::content > [disabled] {
       color: var(--paper-menu-disabled-color, --disabled-text-color);
     }
 
-    :host > ::content > *:focus {
+    .content > ::content > *:focus {
       position: relative;
       outline: 0;
+
+      @apply(--paper-menu-colored-focused-item);
     }
 
-    :host > ::content > *:focus:after {
+    .content > ::content > *:focus:after {
       @apply(--layout-fit);
       background: currentColor;
       /* FIXME move to paper-styles for next widget */
       opacity: 0.12;
       content: '';
 
-      @apply(--paper-menu-focused-item);
+      @apply(--paper-menu-colored-focused-item-after);
     }
 
-    :host > ::content > *[colored]:focus:after {
+    .content > ::content > *[colored]:focus:after {
```

Comment:
Just curious: why did you change this? You mentioned to me before that `:host > ::content` was acceptable.